### ISSUE TITLE
Provide status message for filtered searches

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#10238]
**What**
Filtering search results updates those results. That change needs to be communicated to screen readers e.g. that the number of results has gone down.

**Where**
https://wellcomecollection.org/collections

**TODO**

- [ ] Communicate change of search results after filtering to screen readers via ARIA live property

**Background**

From the Hassell Inclusion accessibility audit August 2023.

Dovetail: https://proficient-wallaby-dpwd.dovetailapp.com/insights/Provide-status-message-for-filtered-searches-5wUuazgUNqrrPPP0XG2kfX
WCAG fail

Miro board: https://miro.com/app/board/uXjVMjsL5YE=/

**Comment by rcantin-w** [comment]
This raised a few more questions as the active filters were being read out as well (and not necessarily in the most helpful way?) so I'm trying to figure out how to refactor that section to make it as useful as possible. [See Slack convo for details](https://wellcome.slack.com/archives/CUA669WHH/p1696513392418849).

**Comment by rcantin-w** [comment]
In prod
